### PR TITLE
Enable fan speed changes from the dashboard

### PR DESCRIPTION
**Pull request by MizterB**
Changing fan speeds from the dashboard calls the "turn_on" service with a "speed" parameter - it does not call the "set_speed" service directly.  This PR modifies "turn_on" to handle the optional speed parameter.

### CODE REVIEW

**Comment by MizterB** [comment]
By the way, apologies for the formatting changes.  I'm not forcing my personal preferences here - these were automatically applied by HA's VSCode devcontainer, so I'm assuming things brings the code into greater alignment with HA's standards.

File: custom_components/bond/fan.py [hunk 1]
```diff
@@ -5,27 +5,25 @@
     SPEED_LOW,
     SPEED_MEDIUM,
     SPEED_HIGH,
-    FanEntity
+    FanEntity,
```
Comment (niemyjski) [inline custom_components/bond/fan.py]: Can you please remove the trailing ,

File: custom_components/bond/fan.py [hunk 2]
```diff
@@ -5,27 +5,25 @@
...
 )
 
-from bond import (
```
Comment (niemyjski) [inline custom_components/bond/fan.py]: Can you please revert all formatting based changes.

**Review by niemyjski — CHANGES_REQUESTED** [review]

File: custom_components/bond/fan.py (hunk 2 above)
Comment (MizterB) [reply custom_components/bond/fan.py]: All set

**Comment by niemyjski** [comment]
Thanks for the pr

**Pull request merged**